Formatting Mailing Address

HamzaIbnFarooq opened the pull request:
#### What are the relevant tickets?
Fixes #1017 

#### What's this PR do?
It formats the mailing address in the footer.

#### How should this be manually tested?
Navigate to any page which shows the footer and mailing address in it.

#### Screenshots:
##### Before (Desktop View):
![image](https://user-images.githubusercontent.com/42243411/94237759-c9583f80-ff28-11ea-9efb-091a67593450.png)
##### After (Desktop View):
![image](https://user-images.githubusercontent.com/42243411/94237820-e12fc380-ff28-11ea-9897-d9f7122a94d7.png)


##### Before (Mobile View):
![image](https://user-images.githubusercontent.com/42243411/94238198-6adf9100-ff29-11ea-9cff-564629c2107e.png)
##### After (Mobile View):
![image](https://user-images.githubusercontent.com/42243411/94238355-b3974a00-ff29-11ea-91e6-898eeb2bd256.png)


### Acceptance criteria

- [ ] screenshots of desktop and mobile view 
- [ ] make sure the change applies to the home page as well as the application page

umarmughal824 reviewed (approved):
LGTM 👍

ahmed-belal commented:
@pdpinch could you confirm the UI change?

pdpinch reviewed (changes requested):
Thanks Hamza. 

On the issue there is an acceptance criteria for desktop and mobile screenshots. Can you add a mobile screenshot so we can see what it looks when the browser window is narrow?

This wasn't part of the issue, but why aren't the three columns of text top-aligned? Is that something you can change?

![image](https://user-images.githubusercontent.com/430126/94177324-5cc93c00-fe67-11ea-9f66-34e5136affcb.png)

HamzaIbnFarooq commented:
Thanks for the review @pdpinch.

I have added mobile screenshots in the description and also changed the alignment of those 3 columns, kindly have a look to verify if it is what we require.

pdpinch reviewed (approved):
Looks good Hamza. 

I don't think we need any more reviews now that @umarmughal824 and I have both reviewed. 

You can merge this. No need to squash, since there is only one commit already. 

If you merge on Monday it should be included in the release candidate Monday afternoon. You'll need to verify the change on the RC server. 
Please

HamzaIbnFarooq commented:
> Looks good Hamza.
> 
> I don't think we need any more reviews now that @umarmughal824 and I have both reviewed.
> 
> You can merge this. No need to squash, since there is only one commit already.
> 
> If you merge on Monday it should be included in the release candidate Monday afternoon. You'll need to verify the change on the RC server.
> Please

Thanks for the explanation, Peter.

I am merging it now.
